modules/ROOT/pages/message-flows.adoc

```diff
@@ -0,0 +1,36 @@
+= Message Flows
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To receive messages from a trading partner, create a message flow and configure the sections for the source, map, and target.
+Your organization is typically referred to as the host. The enterprise or customer you interact with is referred to as your partner.
+
+The source of the message flow is defined by the partner, a receiving endpoint at the host, often an API, and the message type you are receiving from the partner.
+
+The target is the final destination of a message flow and is defined by a host message type and endpoint, often a Host API.
```

Comment:
```suggestion
The target is the final destination of a message flow and is defined by a host message type and endpoint, often a host API.
```

Reply:
Is this "host" API the same API as in the source definition; should that definition include the word "host" too?